api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -583,24 +583,24 @@ public Integer getCountOfUsers(String name, List<Role> roles, boolean includeRet
 		if (name != null) {
 			name = StringUtils.replace(name,", ", " ");
 		}
-		
+
 		if (roles == null) {
 			roles = new ArrayList<>();
 		}
-		
```

Comment:
perfect! exactly right 👍

Reply:
@teleivo Thanks, I will make the necessary changes.